django/db/backends/oracle/utils.py

```diff
@@ -10,16 +10,22 @@ class InsertVar:
     insert statement.
     """
     types = {
+        'AutoField': int,
+        'BigAutoField': int,
+        'IntegerField': int,
+        'BigIntegerField': int,
+        'PositiveSmallIntegerField': int,
+        'PositiveIntegerField': int,
         'FloatField': Database.NATIVE_FLOAT,
         'CharField': str,
         'DateTimeField': Database.TIMESTAMP,
-        'DateField': Database.DATETIME,
+        'DateField': Database.Date,
```

Comment:
Please revert this change.

Reply:
@felixxm the tests fail if I do.

Reply:
That's weird :thinking: Let me check.

Reply:
OK, it's my mistake from #11462 :disappointed: Please keep this change in the main commit and use `Database.DATE`.

Reply:
I put this into a separate commit as well.